example/sampleProp.conf

```diff
@@ -1,6 +1,10 @@
 WebSocketUrl = "Webesocket-URL"
 vROpsUID = "Username"
 vROpsPass = "Password"
+MqttServer = "Mqtt-IP"
+MqttPort = "Mqtt-Port"
```

Comment:
This kind of naming kind of suggests that we plan to use the same MQTT broker for device->gateway and for gateway->DCC communication. Is this our plan?